be/src/exec/vectorized/orc_scanner_adapter.cpp

```diff
@@ -937,8 +937,8 @@ OrcScannerAdapter::OrcScannerAdapter(const std::vector<SlotDescriptor*>& src_slo
           _tzinfo(cctz::utc_time_zone()),
           _tzoffset_in_seconds(0),
           _drop_nanoseconds_in_datetime(false),
-          _broker_load_mode(false),
-          _strict_mode(false),
+          _broker_load_mode(true),
+          _strict_mode(true),
```

Comment:
Why modify default _strict_mode? It's will effect load.

Reply:
It's just a default value.  The user code will set it properly when to use it.

in orc_scanner.cpp
```
    _orc_adapter = std::make_unique<OrcScannerAdapter>(_orc_slot_descriptors);
    _orc_adapter->set_broker_load_mode(_strict_mode);
```

and in hdfs_scanner_orc.cpp

```
    _orc_adapter = std::make_unique<OrcScannerAdapter>(_src_slot_descriptors);
    _orc_row_reader_filter =
            std::make_shared<OrcRowReaderFilter>(_scanner_params, _file_read_param, _orc_adapter.get());
    _orc_adapter->disable_broker_load_mode();
```